README.md

```diff
@@ -145,15 +175,18 @@ Retrieve the current setting for repeat
     - `NO_REPEAT` will not repeat the queue
     - We created constants named `GMusic.Playback.LIST_REPEAT`, `GMusic.Playback.SINGLE_REPEAT`, `GMusic.Playback.NO_REPEAT`
 
-#### `playback.toggleRepeat(mode)`
+#### `playback.setRepeat(mode)`
 Change the current setting for repeat
 
-- mode `String` - Optional mode to change repeat to
-    - If not specified, we will toggle to the next mode
-        - The order is `NO_REPEAT`, `LIST_REPEAT`, `SINGLE_REPEAT`
+- mode `String` - Required mode to change repeat to
```

Comment:
Same feedback as `setShuffle`; please avoid "required" and using names in definitions